devtoolsNG/xen/smoke/index.js

```diff
@@ -0,0 +1,97 @@
+import '../elements/inspector/data-explorer.js';
+
+const {logElt, opsByStoreElt, de} = window; // extract DOM elements
+
+de.object = {Raw: true, RawStoreMessage: true, StoreSyncMessage: true, StoreMessage: true};
+de.addEventListener('object-change', e => {
+  console.log(e);
+});
+
+const dom = (tag, props, parent) => parent.appendChild(Object.assign(document.createElement(tag), props));
+const entry = (object, container) => dom('data-explorer', {object, expand: true}, container || logElt);
+
+let entries;
+
+let i = 0;
+const receive = () => {
+  const raw = entries[i];
+  let data = raw;
+  try {
+    data = JSON.parse(raw);
+  } catch(x) {
+    //
```

Comment:
Empty Comment?

Reply:
Empty catch blocks disallowed by linter. Do you have a preferred format or other solution?